src/pages/workspace/WorkspacesListPage.js

```diff
@@ -122,7 +127,6 @@ class WorkspacesListPage extends Component {
                 brickRoadIndicator: PolicyUtils.getPolicyBrickRoadIndicatorStatus(policy, this.props.policyMembers),
                 pendingAction: policy.pendingAction,
                 isPolicy: true,
-                errors: policy.errors,
```

Comment:
This change caused [a regression](https://github.com/Expensify/App/issues/12452)

Reply:
Didn't see any discussion about this change on this PR. Going to remove it and perform all the testing steps again to see if it can be safely added back

Reply:
We had a discussion on this. We wanted to only show the error message on the workspace details page. Like in this video.

https://user-images.githubusercontent.com/26106822/198982672-521947bb-c0f9-4535-bd69-095ff3de6df5.mp4

Reply:
https://expensify.slack.com/archives/C01GTK53T8Q/p1666955764560879

Reply:
I now understand the context of that thread, and can see that my fix wasn't what was agreed upon in that thread. However, this PR didn't achieve what was agreed upon in that thread either. It was still a regression – we forgot to test the delete workspace flow with this PR. In that case, the workspace was just not showing at all, so there was no way to dismiss the red dot error that appeared in the settings menu.